srv/modules/runners/proposal.py

```diff
@@ -121,6 +121,10 @@ def _parse_args(kwargs):
     """
     args = STD_ARGS.copy()
     args.update(kwargs)
+    if 'kwargs' in kwargs:
+        for key in kwargs['kwargs']:
```

Comment:
Wouldn't it be more efficient to use ```dict.update()``` here? Something like
```
if 'kwargs' in kwargs:
    args.update(args.pop('kwargs')
```

Reply:
Will be circling back soon :)